pleskdistup/actions/emails.py

```diff
@@ -52,3 +54,31 @@ def _revert_action(self) -> action.ActionResult:
 
     def estimate_post_time(self) -> int:
         return 5
+
+
+class RestoreDovecotConfiguration(action.ActiveAction):
+    dovecot_config_path: str
+
+    def __init__(self):
+        self.name = "restore Dovecot configuration"
+        self.dovecot_config_path = "/etc/dovecot/dovecot.conf"
+
+    def _is_required(self) -> bool:
+        return os.path.exists(self.dovecot_config_path)
+
+    def _prepare_action(self) -> action.ActionResult:
+        files.backup_file(self.dovecot_config_path)
+        return action.ActionResult()
+
+    def _post_action(self) -> action.ActionResult:
+        path_to_backup = plesk.CONVERTER_TEMP_DIRECTORY + "/dovecot.conf.bak"
+        if os.path.exists(self.dovecot_config_path):
+            shutil.copy(self.dovecot_config_path, path_to_backup)
+            motd.add_finish_ssh_login_message(f"The dovecot configuration '{self.dovecot_config_path}' has been restored from CentOS 7. Modern configuration was placed in '{path_to_backup}'.")
```

Comment:
Does anything confines this action to CentOS 7 only? If not, propose to drop CentOS 7 mention from the message.

Reply:
I think could be used for other repositories as well, so lets remove mention of centos 7